src/main/java/domain/CarFactory.java

```diff
@@ -0,0 +1,34 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CarFactory {
+    public static final String COMMA = ",";
+    public static final String BLANK = " ";
+    private final List<Car> cars;
+    private final String[] carNames;
```

Comment:
상수와 멤버변수 사이에 공백이 추가되어야합니다 🙂
```suggestion
    public static final String COMMA = ",";
    public static final String BLANK = " ";

    private final List<Car> cars;
    private final String[] carNames;
```